setup.py

```diff
@@ -35,7 +35,7 @@ def get_install_requires() -> List[str]:
         "colorlog",
         "numpy<1.20.0",
         "packaging>=20.0",
-        "scipy!=1.4.0",
+        "scipy!=1.4.0",  # TODO(kstoneriv3) Change to 1.7.0
```

Comment:
Assuming that we release `QMCSampler` after the release of Scipy 1.7.0, does this make sense? Are there a better way of package dependency management?

Reply:
IMO, it's fine to change this to `scipy>=1.7.0`. But, we have to check the current CI works well with `scipy==1.7.0` before merging this PR.

Reply:
OK. I will change this and see if CI works.

Reply:
The CI fails when `scipy>=1.7.0` and it seems that we have to wait until the Scipy 1.7.0 is released to see if it works.

Reply:
The CI fails when `scipy>=1.7.0` and it seems that we have to wait until the Scipy 1.7.0 is released to see if it works.